src/concrete_solve.jl

```diff
@@ -834,7 +913,7 @@ function DiffEqBase._concrete_solve_adjoint(prob, alg, sensealg::ReverseDiffAdjo
 
     function reversediff_adjoint_forwardpass(_u0, _p)
         if (convert_tspan(sensealg) === nothing &&
-            ((haskey(kwargs, :callback) && has_continuous_callback(kwargs[:callback])))) ||
+            ((haskey(kwargs, :callback) && has_a_callback(kwargs[:callback])))) ||
```

Comment:
was this a mistake?

Reply:
yeah, pressed `a` accidentally, thought I corrected it, but it crept in